doc/14-features.md

```diff
@@ -390,12 +397,13 @@ escape characters when followed by a space or comma, but cannot be escaped thems
 As a result all trailling slashes in these fields are replaced with an underscore.  This
 predominantly affects Windows paths e.g. `C:\` becomes `C:_`.
 
-The database is assumed to exist so this object will make no attempt to create it currently.
+The database/bucket is assumed to exist so this object will make no attempt to create it currently.
 
 If [SELinux](22-selinux.md#selinux) is enabled, it will not allow access for Icinga 2 to InfluxDB until the [boolean](22-selinux.md#selinux-policy-booleans)
 `icinga2_can_connect_all` is set to true as InfluxDB is not providing its own policy.
 
-More configuration details can be found [here](09-object-types.md#objecttype-influxdbwriter).
+More configuration details can be found [here](09-object-types.md#objecttype-influxdbwriter)
+and [here](09-object-types.md#objecttype-influxdb2writer).
```

Comment:
Having two links labeled "here" is of limited usefulness. The link text should state the version.